readthedocs/rtd_tests/tests/test_doc_building.py

```diff
@@ -30,7 +30,7 @@ class TestLocalEnvironment(TestCase):
     def setUp(self):
         self.project = Project.objects.get(slug='pip')
         self.version = Version(slug='foo', verbose_name='foobar')
-        self.project.versions.add(self.version)
+        self.project.versions.add(self.version, bulk=False)
```

Comment:
This is what happened before my change:

```
________________________________________________________________________________________________ TestLocalEnvironment.test_normal_execution _________________________________________________________________________________________________

self = <readthedocs.rtd_tests.tests.test_doc_building.TestLocalEnvironment testMethod=test_normal_execution>

    def setUp(self):
        self.project = Project.objects.get(slug='pip')
        self.version = Version(slug='foo', verbose_name='foobar')
>       self.project.versions.add(self.version)

rtd_tests/tests/test_doc_building.py:33: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <django.db.models.fields.related_descriptors.RelatedManager object at 0x7fc67575de90>, objs = [<Version: Version foobar of Pip (None)>], kwargs = {}, bulk = True, db = 'default'
check_and_update_obj = <function check_and_update_obj at 0x7fc675c55230>, pks = [], obj = <Version: Version foobar of Pip (None)>

    def add(self, *objs, **kwargs):
        bulk = kwargs.pop('bulk', True)
        objs = list(objs)
        db = router.db_for_write(self.model, instance=self.instance)
    
        def check_and_update_obj(obj):
            if not isinstance(obj, self.model):
                raise TypeError("'%s' instance expected, got %r" % (
                    self.model._meta.object_name, obj,
                ))
            setattr(obj, self.field.name, self.instance)
    
        if bulk:
            pks = []
            for obj in objs:
                check_and_update_obj(obj)
                if obj._state.adding or obj._state.db != db:
                    raise ValueError(
                        "%r instance isn't saved. Use bulk=False or save "
>                       "the object first." % obj
                    )
E                   ValueError: <Version: Version foobar of Pip (None)> instance isn't saved. Use bulk=False or save the object first.

../.tox/py27/lib/python2.7/site-packages/django/db/models/fields/related_descriptors.py:568: ValueError
```

Bulk behavior of add() method of related managers was changed in Django 1.9. To revert the behavior of the add() method in Django 1.8, I added `bulk=False`.

Here is some documentation related to the change in Django 1.9: https://docs.djangoproject.com/en/dev/releases/1.9/#bulk-behavior-of-add-method-of-related-managers

I think it would be better to change it to this:
```python
self.version = self.project.versions.create(slug='foo', verbose_name='foobar')
```